dspot-web/README.md

```diff
@@ -0,0 +1,45 @@
+# Introduction
+
+This is a prototype web interface of Dspot, inspired from [CommitGuru](http://commit.guru/).
+
+Prototype demo link: <http://35.246.191.225:3000>
+
+Author: Henry Luong
+
+## Running
+
+```
+cd dspot-web
+npm install --no-optional
+npm run-script start
+```
+
+## Screenshots
+
+### Front page
+
+The front page enables to submit a new repo.
+
+![Pic 1](https://github.com/Tailp/dspot/blob/web/dspot-web/screenshots/pic1.png)
```

Comment:
merging means a single screenshot with the complete window, not two concatenated windows :-)

Reply:
Ah, I see. I'll do that, but you'll have to zoom in later since I need to zoom out to capture the whole screen :).